tests/test_approve.py

```diff
@@ -427,22 +462,25 @@ def f_osc_core(*_args: Any, **_kwds: Any) -> NoReturn:
 @responses.activate
 @pytest.mark.parametrize("fake_qem", ["NoResultsError isn't raised"], indirect=True)
 @pytest.mark.usefixtures("fake_qem", "fake_two_passed_jobs", "f_osconf")
-def test_osc_unknown_exception(caplog: LogCaptureFixture, monkeypatch: MonkeyPatch) -> None:
+def test_osc_unknown_exception(caplog: LogCaptureFixture, monkeypatch: pytest.MonkeyPatch) -> None:
     caplog.set_level(logging.DEBUG, logger="bot.approver")
 
     def f_osc_core(*_args: Any, **_kwds: Any) -> NoReturn:
-        raise Exception("Fake OBS exception")
+        class ArbitraryObsException(Exception):
+            def __init__(self) -> None:
+                super().__init__("Fake OBS exception")
+
+        raise ArbitraryObsException
```

Comment:
This goes against the intention of the style check. Now one can still not catch this exception without a generic exception handler as `ArbitraryObsException` is probably not accessible from outside this function. (Or is it accessible? Then this would be ok of course.)